web/templates/note/index.html.eex

```diff
@@ -1,10 +1,19 @@
 <h1>Listing Notes</h1>
 
+<p><%= link "New note", to: note_path(@conn, :new) %></p>
+
 <ul>
   <%= for note <- @notes do %>
     <li>
-      <div><b><%= note.title %></b></div>
-      <div><%= note.description %></div>
+
+      <div>
+        <b> <%= link note.title, to: note_path(@conn, :show, note.id) %> </b> (<%= note.id %>)
```

Comment:
This white space between the `<b>` tags is a little weird. I'd remove it. If you want to space it out in the HTML, I'd put the link on it's own line.